### ISSUE TITLE
When using JobStore, I found that the trigger will be deleted from the data table after the trigger is completed. Is there any way to stop him?

### ISSUE THREAD

**Issue by JL-Code** [quartznet/quartznet#654]
When using JobStore, I found that the trigger will be deleted from the data table after the trigger is completed. Is there any way to stop him?

**Comment by JL-Code** [comment]
I found a QRTZ_FIRED_TRIGGERS data table with no data at all. Is the data in this table added to QRTZ_FIRED_TRIGGERS after the Trigger trigger is completed?

**Comment by ronnymgm** [comment]
We faced the same inconvenient. We want to keep in DB history of all triggers executed. What we did was modify the StdAdoDelegate with a new table QRTZ_FIRED_TRIGGERS_HIST and methods to save a record each time a trigger completes execution. We couldn't find an easy way to achieve this with a plugin (version 2x), so we ended modifying and building the Quartz solution with our changes (not the best solution honestly).
